elastic/beats
# Add `task_manager_utilization` metricset to Kibana metricbeat module

This PR adds a new metricset to the Metricbeat `kibana` module. This collects the background worker utilization load from the API added in https://github.com/elastic/kibana/pull/153600. We plan to use this metric for autoscaling in serverless.

## Checklist


## How to test this PR locally

- run kibana from source & ES from snapshot
- pull down this PR, `cd metricbeat`
- `mage build`
- `./metricbeat modules enable kibana-xpack`
- uncomment the `elasticsearch.output` username/password lines in `metricbeat.yml` (`elastic:changeme`)
- update `./modules.d/kibana-xpack.yml`:
```yml
- module: kibana
  xpack.enabled: false
  period: 10s
  hosts: ["localhost:5601"]
  basepath: "abc" # whatever your base path is
  username: "elastic"
  password: "changeme"
```
- `./metricbeat -e`
- create `metricbeat-*` data view in kibana
- you should now be seeing some log entries which contain `task_manager_utilization.load`
